MmSupervisorPkg/MmSupervisorPkg.dec

```diff
@@ -37,6 +37,7 @@
   gMmProtectedRegionHobGuid                       = { 0x6c0792ac, 0x13d7, 0x431b, { 0xa4, 0x89, 0x3, 0x2f, 0x4a, 0xf9, 0x73, 0x80 } }
   gMmSupervisorPolicyFileGuid                     = { 0x81ff0793, 0x3e18, 0x489b, { 0x9a, 0x8, 0xa1, 0xeb, 0x71, 0xb2, 0x3d, 0x20 } }
   gMmSupervisorDriverDispatchGuid                 = { 0x2e135da6, 0xade0, 0x4b96, { 0x9b, 0x40, 0xb2, 0xf3, 0x67, 0xe9, 0xf7, 0xf7 } }
+  gMmSupervisorCoreGuid                           = { 0x4e4c89dc, 0xa452, 0x4b6b, { 0xb1, 0x83, 0xf1, 0x6a, 0x2a, 0x22, 0x37, 0x33 } }

```

Comment:
Please add a comment to describe the value of this GUID

Reply:
Updated.